tests/testthat/test-TextType.R

```diff
@@ -49,3 +49,25 @@ testthat::test_that(
     unlink(out)
   }
 )
+
+
+
+testthat::test_that(
+  "We can parse a richer abstract into markdown",
+  {
+    skip_on_cran()
+    skip_on_appveyor()
+    skip_on_os("windows")
```

Comment:
Looks like I must have deleted this test and it's being added back in?  Not sure why I would have removed it in the first place.  Only thought is can you try removing the `skip_on_appveyor()` and `skip_on_os("windows")` directives and see if check still passes, since I think we have resolved that issue?